src/qt/qt_mainwindow.hpp

```diff
@@ -33,6 +33,7 @@ class MainWindow : public QMainWindow
     void blitToWidget(int x, int y, int w, int h, int monitor_index);
     QSize getRenderWidgetSize();
     void setSendKeyboardInput(bool enabled);
+    void setUiPauseState(bool paused);
```

Comment:
This should be made a signal since Qt dislikes UI modifications from non-UI threads.

Reply:
Ah, did not realize it was a different thread. Thanks. I'll redo as a signal (I now see how similar signals are implemented elsewhere in `qt_platform.cpp` and called with `QTimer::singleShot`).